src/Blocks/Heading3.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Notion\Blocks;
+
+use Notion\Common\RichText;
+
+/**
+ * @psalm-import-type BlockJson from Block
+ * @psalm-import-type RichTextJson from \Notion\Common\RichText
+ *
+ * @psalm-type Heading3Json = array{
+ *      heading_3: array{
+ *          text: RichTextJson[],
+ *      },
+ * }
+ */
+class Heading3 implements BlockInterface
```

Comment:
The fact that there's H1, H2, H3, etc. makes me think that perhaps a simpler interface to use most of this stuff could really just be a Markdown->Notion API tool, perhaps?

Then the entire Blocks/* concept could perhaps be `@internal`, mostly.

Reply:
I am thinking on develop a parser from Markdown to Notion API blocks (and vice-versa). May be the next big step.